script/swap_and_bridge/contracts/SwapAndBridge_integration_test.s.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.23;
+
+import { Script, console2 } from "forge-std/Script.sol";
+import { SwapAndBridge } from "src/L1/SwapAndBridge.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/// @title IWrappedETH - Wrapped Ether Token interface
+/// @notice This contract is used to wrap the a LST.
+interface IWrappedETH is IERC20 {
+    receive() external payable;
+}
+
+/// @title TestIntegrationScript
+/// @notice This contract is used to run integration test for the SwapAndBridge contract.
+contract TestIntegrationScript is Script {
+    // SwapAndBridge contract
+    SwapAndBridge swapAndBridge;
+
+    // The L1 LST token
+    IWrappedETH l1LSTToken;
+
+    // Address used for E2E tests
+    address constant testAccount = address(0xc0ffee);
+
+    function setUp() public {
+        vm.deal(testAccount, 500000 ether);
+    }
+
+    function runMinToken() public {
+        console2.log("Testing minL1TokensPerETH...");
+        // The conversion rate is 1 ETH = 1e18 LST.
+        // Any value of minL1TokensPerETH larger than 1e18 will revert the transaction.
+        vm.startPrank(testAccount);
+
+        console2.log("Testing no minimum...");
+        swapAndBridge.swapAndBridgeToWithMinimumAmount{ value: 1 ether }(testAccount, 0);
```

Comment:
Should we add some `require` after its execution to test if this function was really executed as expected?

Reply:
added